### PR TITLE
*: Misc refactors, and doc additions

### PR DESCRIPTION
**Pull request by derekparker**
Refactors some code, adds a bunch of docstrings and just generally fixes
a bunch of linter complaints.

### CODE REVIEW

File: pkg/proc/variables.go [hunk 1]
```diff
@@ -1026,13 +1031,13 @@ func (v *Variable) loadValueInternal(recurseLevel int, cfg LoadConfig) {
 //   is performed.
 // * If srcv and dstv have the same type and are both addressable then the
 //   contents of srcv are copied byte-by-byte into dstv
-func (dstv *Variable) setValue(srcv *Variable, srcExpr string) error {
+func (v *Variable) setValue(srcv *Variable, srcExpr string) error {
```
Comment (aarzilli) [inline pkg/proc/variables.go]: I would prefer that this remained `dstv` for symmetry with `srcv`.

File: service/debugger/debugger.go [hunk 2]
```diff
@@ -402,6 +405,7 @@ func (d *Debugger) AmendBreakpoint(amend *api.Breakpoint) error {
 	return copyBreakpointInfo(original, amend)
 }
 
+// CancelNext will clear internal breakpoints, thus cancelling the 'next' operation.
```
Comment (aarzilli) [inline service/debugger/debugger.go]: I'd say "cancelling the 'next', 'step' or 'stepout' operation"

File: pkg/proc/variables.go (hunk 1 above)
Comment (derekparker) [reply pkg/proc/variables.go]: I'll change if you feel strongly, but I like the consistency of receiver names for these methods.

Comment (aarzilli) [reply pkg/proc/variables.go]: ok, leave it

**Comment by derekparker** [comment]
@aarzilli done and updated.

**Pull request merged**